frontend/js/pages/BattleDetail.js

```diff
@@ -43,33 +44,43 @@ function BattleDetail(props) {
   if (error) {
     return 'Ocurred an error';
   }
-  if (!battle) {
+  if (!user) {
```

Comment:
Be careful with these binary operations in JS, whenever possible use `lodash` methods as they are more readable and more precise on behavior (isEmpty, isNull, isUndefines, ...)

Reply:
I'll keep that in mind. Thanks!!!